src/data/ExplorePageData.ts

```diff
@@ -1,29 +1,107 @@
 import { Theme } from '@/theme/Theme';
-import { GreaterMattapanZoning as GreaterMattapanZoningData } from '@/data/GreaterMattapanZoning';
-import { BlueHillAveCorridor as BlueHillAveCorridorData } from '@/data/BlueHillAveCorridor';
 
-export const ExplorePageData: MapGeoJsonData[] = [
+export const EXPLORE_PAGE_BOUNDARY_DATA: GeoJsonMeta[] = [
```

Comment:
This whole data structure is really quite temporary and needs more alignment with Powerful Pathways to determine how they will handle their data (Google sheet API?)